pkg/operator2/idp.go

```diff
@@ -233,16 +233,7 @@ func (c *authOperator) convertProviderConfigToIDPData(providerConfig *configv1.I
 	return data, nil
 }
 
-func (c *authOperator) discoverOpenIDURLs(issuer, key string, ca configv1.ConfigMapNameReference, fallbackURLs configv1.OpenIDURLs) (*osinv1.OpenIDURLs, error) {
-	// TODO drop after 4.0beta3
-	if len(issuer) == 0 {
-		return &osinv1.OpenIDURLs{
-			Authorize: fallbackURLs.Authorize,
-			Token:     fallbackURLs.Token,
-			UserInfo:  fallbackURLs.UserInfo,
-		}, nil
-	}
-
```

Comment:
I _think_ this should be good now, since we're beta4

Reply:
I believe this is fine now.